core/src/main/java/org/bitcoinj/wallet/Wallet.java

```diff
@@ -230,8 +230,6 @@ protected boolean removeEldestEntry(Map.Entry<Sha256Hash, Transaction> eldest) {
     // that was created after it. Useful when you believe some keys have been compromised.
     private volatile long vKeyRotationTimestamp;
 
-    protected CoinSelector coinSelector = DefaultCoinSelector.get();
```

Comment:
I would consider leaving this and making it `final` -- this will allow subclasses and tests more flexibility in overriding.

Reply:
That's a good idea. I'm leaving it in.